docs/start/myths/overlays.md

```diff
@@ -5,16 +5,52 @@ parent: Debunking myths
 nav_order: 3
 ---
 
-# Overlays
-
-{: .warning }
-**TODO:**
-Starter content. This content still needs to be written.
-
-- what is an overlay
-- to consider: WP plugins for overlays and their claim (maybe not) .
-- the difference between an overlay and a toolbar.
-- link to [overlayfactsheet.com](https://overlayfactsheet.com/en/)
-- include video [Overlays Underwhelm](https://www.youtube.com/live/RDkqJaBwgKU?si=4K5oAYp0ttz7jmNj) by Adrian Roselli.
-- include video [Overlays Underwhelm](https://www.youtube.com/watch?v=fyRxd072JrA) by Léoni Watson.
-- add more resources from within the WP community.
+# Accessibility Overlays: A Comprehensive Overview
+
+### What are Accessibility Overlays?
```

Comment:
Change into an H2.
Please read [Accessible heading structure](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/) that explains why.